README.md

```diff
@@ -121,103 +85,41 @@ To extract data from this application:
 ```bash
 $ ./bin/nrjmx -hostname localhost -port 1689 -uriPath "org.opends.server.protocols.jmx.client-unknown" -username user -password pwd
 ```
+### Troubleshooting
 
----
```

Comment:
@ramonguiu Same here, before we decide whether to move this to the docs site or not.

Reply:
Ah, you moved this to TROUBLESHOOT.MD?